code/modules/mob/living/emote.dm

```diff
@@ -107,9 +107,14 @@
 		message_animal_or_basic = custom_message
 	. = ..()
 	message_animal_or_basic = initial(message_animal_or_basic)
+	var/mob/living/carbon/carbon_user = user
+	if(!user.can_speak() || user.oxyloss >= 50)
+		return //stop the sound if oxyloss too high/cant speak
+	if(istype(carbon_user)) // For masks that give unique death sounds
+		if(isclothing(carbon_user.wear_mask) && (carbon_user.wear_mask.unique_death))
+			playsound(carbon_user, carbon_user.wear_mask.unique_death, 200, TRUE, TRUE)
+			return
 	if(. && user.death_sound)
```

Comment:
You should be respecting the return value of the parent call, as is being done here. 
IE, just put `!.` in your `can_speak` line.

Reply:
I'm kinda confused by this.
Did I do it right now?